c/secp256k1_ripemd160_sha256_sighash_all.c

```diff
@@ -0,0 +1,190 @@
+#include "sha256.h"
```

Comment:
Please add some comment about how to use the script in CKB. What should be put in script args, and what are expected in the witness.